imperial_coldfront_plugin/views.py

```diff
@@ -539,6 +540,28 @@ def add_rdf_storage_allocation(request):
         return HttpResponseForbidden()
 
     if request.method == "POST":
+        filesystem_name = request.POST.get("filesystem_name")
+        fileset_name = request.POST.get("fileset_name")
+        owner_id = request.POST.get("owner_id")
+        group_id = request.POST.get("group_id")
+        path = request.POST.get("path")
+        permissions = request.POST.get("permissions")
+        block_quota = request.POST.get("block_quota")
+        files_quota = request.POST.get("files_quota")
```

Comment:
So these values should come from a variety of places:
- filesystem_name - should be a setting
- fileset_name - will use the value of `group_id` below
- owner_id and group_id - will be "root" for now
- path - should be a prefix (given by a setting) with `group_id` appended
- permissions - should be a setting - "700" is a sensible guess
- files_quota - should be a setting - 1000 is ok for development
- block_quota - will use the value from the form accessible as `form.cleaned_data["size"]` after `form.is_valid()` below.